clang/test/SemaCXX/cxx2c-enum-compare.cpp

```diff
@@ -1,9 +1,10 @@
-// RUN: %clang_cc1 %s -std=c++2c -fsyntax-only -verify -triple %itanium_abi_triple
+// RUN: %clang_cc1 %s -std=c++2c -fsyntax-only -verify=both,expected
+// RUN: %clang_cc1 %s -std=c++2c -fsyntax-only -verify=both,nowarn -Wno-enum-enum-conversion
```

Comment:
`nowarn` is unused, but I think it should be silencing the invalid comparison diagnostic, shouldn't it?

Reply:
Comparing with double seems a lot weirder than comparing with another enum and no one complained about that; I think it should remain an error

Reply:
Whoops, I was misreading the test, there is test coverage for this. Sorry!